cypress/integration/vizsurvey.spec.js

```diff
@@ -154,6 +154,7 @@ describe("vizsurvey", () => {
     cy.get("button").contains("Next").click();
     cy.get("#laterAmount").click();
     cy.get("button").contains("Next").click();
+    cy.get("button").contains("Next").click(); // TODO: fix later
```

Comment:
Double next click is needed because we check status after dispatching the last write answer. So the first next click actually sets the status to questionaire and the second next click will navigate to the questionaire. Instead we will have to check if we are on the last question earlier and update the status then.
```
s/c/BarChart.js
358           <Button
359             variant="contained"
360             color="secondary"
361             disableRipple
362             disableFocusRipple
363             style={styles.button}
364             onClick={() => {
365               dispatch(
366                 answer({
367                   choice: choice,
368                   choiceTimestamp: dateToState(DateTime.utc()),
369                 })
370               );
371               if (status === StatusType.Questionaire) {
372                 navigate("/questionaire");
373               }
374             }}
```

Reply:
resolved in https://github.com/The-Discounters/vizsurvey/pull/51/commits/f955a1af2ab8e1173d57d80670a48a0b8ebfaf8f